jujugui/static/gui/src/app/models/legacy-handlers.js

```diff
@@ -299,9 +318,22 @@ YUI.add('juju-legacy-delta-handlers', function(Y) {
       @return {undefined} Nothing.
      */
     annotationInfo: function(db, action, change) {
+      var cleanUpEntityTags = function(tag) {
+        var result = tag.replace(/^(service|unit|machine|environment)-/, '');
```

Comment:
This looks like the second time we've done the cleanup code; this seems like another candidate for being moved to an external utils file to be shared between these two instances, unless there's a difference I'm missing?